setup.py

```diff
@@ -1,41 +1,53 @@
-from setuptools import setup, find_packages
+"""Package setup."""
+from setuptools import setup
+from setuptools import find_packages
 import os
 
 version = '1.0'
 
-setup(name='wildcard.lockdown',
-      version=version,
-      description="",
-      long_description=open("README.txt").read() + "\n" +
-                       open(os.path.join("docs", "HISTORY.txt")).read(),
-      # Get more strings from
-      # http://pypi.python.org/pypi?%3Aaction=list_classifiers
-      classifiers=[
-        "Framework :: Plone",
-        "Programming Language :: Python",
-        ],
-      keywords='',
-      author='',
-      author_email='',
-      url='http://svn.plone.org/svn/collective/',
-      license='GPL',
-      packages=find_packages(exclude=['ez_setup']),
-      namespace_packages=['wildcard'],
-      include_package_data=True,
-      zip_safe=False,
-      install_requires=[
-          'setuptools',
-          'plone.app.registry'
-      ],
-      extras_require={
-          'test': [
-              'plone.app.testing'
-          ]
-          },
-      entry_points="""
-      # -*- Entry points: -*-
-
-      [z3c.autoinclude.plugin]
-      target = plone
-      """
-      )
+setup(
+    name='wildcard.lockdown',
+    version=version,
+    description=(
+        'Plone add-on to be able to make your site read-only except for a set '
+        'of conditions that can be defined and enabled/disabled.'
+    ),
+    long_description=(
+        open('README.rst').read() + '\n' +
+        open(os.path.join('docs', 'HISTORY.rst')).read()
```

Comment:
Rather than docs/HISTORY.rst the change log file is usually at the top level and named CHANGES.rst, as you are moving it on the last commit, please rename it to this other one 🙂